adapter/src/main/java/org/jvalue/ods/adapterservice/datasource/rest/v1/DatasourceEndpoint.java

```diff
@@ -18,10 +22,12 @@
 public class DatasourceEndpoint {
 
     private final DatasourceManager datasourceManager;
+    private final AdapterFactory adapterFactory;
```

Comment:
Is there a reason you add the adapterFactory? I think it can be removed since it is not used.

Reply:
Forgot to remove it, after refactoring. I pushed the change.